source/write/update.txt

```diff
@@ -0,0 +1,251 @@
+a parameter that sets options to configure the update operation.
+If you don't specify any options, the driver performs update
+operations with default settings.
+
+The following table describes the setter methods that you can use to
+configure an ``UpdateOptions`` instance:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``upsert()``
+     - | Specifies whether the update operation performs an upsert operation if no 
+         documents match the query filter. For more information, see the :manual:`upsert
+         statement </reference/command/update/#std-label-update-command-upsert>`
+         in the {+mdb-server+} manual.
+       | Defaults to ``false``
+
+   * - ``bypassDocumentValidation()``
+     - | Specifies whether the update operation bypasses document validation. This lets you 
+         update documents that don't meet the schema validation requirements, if any 
+         exist. For more information about schema validation, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the MongoDB
+         Server manual.
+       | Defaults to ``false``.
+
+   * - ``collation()``
+     - | Specifies the kind of language collation to use when sorting
+         results. For more information, see :manual:`Collation </reference/collation/#std-label-collation>`
+         in the {+mdb-server+} manual.
+
+   * - ``arrayFilters()``
+     - | Provides a list of filters that you specify to select which
+         array elements the update applies to.
+
+   * - ``hint()``
+     - | Sets the index to use when matching documents. 
+         For more information, see the :manual:`hint statement </reference/command/update/#std-label-update-command-hint>`
+         in the {+mdb-server+} manual.
+
+   * - ``let()``
+     - | Provides a map of parameter names and values to set top-level
+         variables for the operation. Values must be constant or closed
+         expressions that don't reference document fields. For more information,
+         see the :manual:`let statement
+         </reference/command/update/#std-label-update-let-syntax>` in the
+         {+mdb-server+} manual.
+
+   * - ``comment()``
+     - | Sets a comment to attach to the operation. For more
+         information, see the :manual:`update command
+         fields </reference/command/update/#update>` guide in the
+         {+mdb-server+} manual for more information.
+
+Modify Update Example
+`````````````````````
+
+The following code uses the ``updateOne()`` method to match documents
+in which the ``name`` field value is ``"Sunrise Pizzeria"``. It then
+sets the ``borough`` field value in the first matching document to
+``"Queens"`` and the ``cuisine`` value to ``"Italian"``.
+
+Because the ``upsert`` option is set to ``true``, the driver inserts a
+new document that has the fields and values specified in the update
+document if the query filter doesn't match any existing documents.
```

Comment:
```suggestion
Because the ``upsert`` option is set to ``true`` in the ``UpdateOptions`` instance,
the driver inserts a new document that has the fields and values specified in the filter
and update document if the query filter doesn't match any existing documents.
```